activerecord/lib/active_record/errors.rb

```diff
@@ -178,10 +178,9 @@ class Rollback < ActiveRecordError
   class DangerousAttributeError < ActiveRecordError
   end
 
-  UnknownAttributeError = ActiveSupport::Deprecation::DeprecatedConstantProxy.new( # :nodoc:
-    'ActiveRecord::UnknownAttributeError', 
-    'ActiveModel::AttributeAssignment::UnknownAttributeError'
-  )
+  # Raised when unknown attributes are supplied via mass assignment.
+  class UnknownAttributeError < ActiveModel::UnknownAttributeError
```

Comment:
Aliasing is a little confusing because you would expect `error.class.to_s` to return `ActiveRecord` exception but it is not. It will cause confusion at least in all bug notification services. I vote for deprecation.